showcase.proto

```diff
@@ -22,43 +22,44 @@ import "google/rpc/status.proto";
 
 package google.showcase.v1;
 
+
 // A service to showcase Generated API Client features and common API patterns
 // used by Google.
 service Showcase {
   // A method used to show unary methods. This method will return the message
   // that was given.
-  rpc Echo(EchoRequest) returns (EchoResponse) {}
+  rpc Echo(EchoRequest) returns (EchoResponse);
 
   // A method used to show server-side streaming methods. This method will
   // split the given content into words and will pass each word back through
   // the stream.
-  rpc Expand(ExpandRequest) returns (stream EchoResponse) {}
+  rpc Expand(ExpandRequest) returns (stream EchoResponse);
```

Comment:
Despite the duplication, can there be a separate `ExpandResponse`? I understand why you did this, but our general guidance is a unique request/response for most calls, and since this may be many people's first exposure, I think this detail matters.

Reply:
(This applies throughout these.)